<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color: #000046
}


QWidget
{
	border-radius: 10px;
}

QLable
{
	color: white;
}

QPushButton
{
	background-color: Black;
	color: white;
	border-radius: 5px;
}

QPushButton::hover
{
	background-color: rgba(63, 94, 251, 255);
	color: white;
	border-radius: 5px;
	min-width: 40ex;
	min-height: 10ex;
}

QRadioButton
{
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>7</y>
      <width>516</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(142, 45, 226, 255), stop:1 rgba(74, 0, 224, 255));
	border-radius: 5px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="exitButton">
        <property name="maximumSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Закрыть</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(238, 9, 121, 255), stop:1 rgba(255, 106, 0, 255));
	border-radius: 8px;
}
QPushButton::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(238, 9, 121, 150), stop:1 rgba(255, 106, 0, 150));
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="minButton">
        <property name="maximumSize">
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Свернуть</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 255), stop:1 rgba(120, 255, 214, 255));
	border-radius: 8px;
}
QPushButton::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 150), stop:1 rgba(120, 255, 214, 150));
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>minButton</zorder>
     <zorder>exitButton</zorder>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>33</y>
      <width>509</width>
      <height>462</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(15, 12, 41, 255), stop:0.492537 rgba(48, 43, 99, 255), stop:1 rgba(36, 36, 62, 255));
	border-radius: 5px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>501</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane
{
	border: 1px;
	border-radius: 5px;
	color: white;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(142, 45, 226, 255), stop:1 rgba(74, 0, 224, 255))
}

QTabBar::tab
{
	background: rgba(74, 0, 224, 255);
	min-width: 40ex;
	min-height: 10ex;
	margin-left: 5px;
	margin-bottom: 5px;
	border-radius: 5px;
	left: -5px;
	color: white
}

QTabBar::tab:selected
{
	background: rgba(142, 45, 226, 255)
}

QLabel
{
	color: black;
}</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="WeatherTab">
     <attribute name="title">
      <string>Погода</string>
     </attribute>
     <widget class="QLabel" name="city_nameLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>549</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>В городе сейчас</string>
      </property>
     </widget>
     <widget class="QLabel" name="degreesLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>15*</string>
      </property>
     </widget>
     <widget class="QLabel" name="weather_typeLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>110</y>
        <width>545</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Облачно</string>
      </property>
     </widget>
     <widget class="QLabel" name="feels_likeLabel">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>130</y>
        <width>545</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ощущается как</string>
      </property>
     </widget>
     <widget class="QLabel" name="Image">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Рисунок</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="favouriteButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>370</y>
        <width>151</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить в избранное</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="city_nameLine">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
	border-radius: 5px;
}</string>
      </property>
      <property name="placeholderText">
       <string>Напишите название города, например &quot;Москва&quot;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>399</x>
        <y>30</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Найти город</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>271</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="wind_speedLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Скорость ветра</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="humidityLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Влажность воздуха</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="visibilityLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Видимость</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="pressureLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Давление</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="wind_speedImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ветер</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="visibilityImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Глаз</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="pressureImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Давл</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="humidityImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Капелька</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="errorLabel">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>102</y>
        <width>361</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добро пожаловать!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="InfoTab">
     <attribute name="title">
      <string>Доп. Информация</string>
     </attribute>
     <widget class="QLabel" name="sunriseLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>23562 seth</string>
      </property>
     </widget>
     <widget class="QLabel" name="sunsetLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>23562 seth</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>275</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QLabel" name="day5_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="day3_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="day7_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="day4_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="day6_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="day2_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="day1_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="temp1_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="temp2_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="temp3_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="temp4_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="temp5_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="temp6_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="temp7_label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="forecastLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Прогноз на 7 дней</string>
      </property>
     </widget>
     <widget class="QLabel" name="Image1">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>40</y>
        <width>31</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Image2">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>80</y>
        <width>31</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Image3">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>120</y>
        <width>31</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Image4">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>160</y>
        <width>31</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Image5">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>200</y>
        <width>31</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Image6">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>240</y>
        <width>31</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Image7">
      <property name="geometry">
       <rect>
        <x>339</x>
        <y>280</y>
        <width>31</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FavoritesTab">
     <attribute name="title">
      <string>Избранное</string>
     </attribute>
     <widget class="QTableView" name="FavoriteTableView">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>11</y>
        <width>491</width>
        <height>401</height>
       </rect>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>245</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="CustomTab">
     <attribute name="title">
      <string>Кастомизация</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>160</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="celsius_radioButton">
         <property name="text">
          <string>Метрическая система</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="fahrenheit_radioButton">
         <property name="text">
          <string>Имперская система</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="kelvin_radioButton">
         <property name="text">
          <string>Стандартная система</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="background_colorButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>50</y>
        <width>299</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать цвет фона</string>
      </property>
     </widget>
     <widget class="QPushButton" name="colorButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>299</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать цвет кнопок</string>
      </property>
     </widget>
     <widget class="QPushButton" name="font_colorButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>130</y>
        <width>299</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать цвет текста</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fontLine">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Выбрать шрифт...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fontButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>131</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Поставить шрифт</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
